timeago/lib/src/timeago.dart

```diff
@@ -51,7 +48,7 @@ void setLocaleMessages(String locale, LookupMessages lookupMessages) {
 /// - If [allowFromNow] is passed, format will use the From prefix, ie. a date
 ///   5 minutes from now in 'en' locale will display as "5 minutes from now"
 String format(DateTime date,
-    {String locale, DateTime clock, bool allowFromNow}) {
+    {String? locale, DateTime? clock, bool? allowFromNow}) {
```

Comment:
I think we can use non nullable with default values for `locale` and `allowFromNow`

```dart
format(DateTime date, {String locale = _default, DateTime? clock, bool allowFromNow = false}
```

Reply:
`_default` can't be used as default value, because it's not constant. `allowFromNow` works, I just changed it!